Added IE Specific Prefix style rule to fix Placeholder color (#2351)
cerner/terra-core: packages/terra-form-input/CHANGELOG.md

Unreleased
----------
### Fixed

> * Fixed Placeholder color for IE by adding vendor specific style rule

Reviewer: Maybe consider updating this to something that mentions the issue - 

`Faded the placeholder text on IE11 by adding vendor specific style rule.`

Author: ec4b0bd518032515927780abeb5ae72600633807

> * Faded the placeholder text on IE11 by adding vendor specific style rule.

Reviewer 2: nit, can we switch this sentence
```suggestion
* Add vendor style rule for IE11 to fade placeholder text.
```

Author: changed change logs for all three components : https://github.com/cerner/terra-core/commit/88ec784c6421f99916bf8abcc9723556f18b0bbb
